src/TxHistory/BalanceBanner.tsx

```diff
@@ -41,8 +41,8 @@ export const BalanceBanner = () => {
             loading={{size: 'small'}}
             error={{
               size: 'inline',
-              fallback: () => (
-                <Text style={styles.pairedBalanceText} testID="pairedTotalText">
+              fallback: ({resetErrorBoundary}) => (
+                <Text onPress={resetErrorBoundary} style={styles.pairedBalanceText} testID="pairedTotalText">
```

Comment:
use a button

Reply:
@jemubm check with the Design if we can add a refresh button at the end of the sentence (Icon.ArrowUpAndDown) or something alike.

Reply:
👍🏻